### DISCUSSION TITLE
gradlew build is failing

### DISCUSSION THREAD

**Discussion by jagdishjadeja** [codemagic-ci-cd/codemagic-docs#2025] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

workflows:
    sample-workflow:
        name: Codemagic Sample Workflow
        max_build_duration: 120
        instance_type: mac_mini_m1
        scripts:
          - name: Build Android APK
            working_directory: android
            script: ./gradlew assembleRelease
        artifacts:
          - app/build/outputs/**/**/*.apk
        publishing:
          email:
            recipients:
              - jagdish.jadeja@formics.io
        triggering:
          events:
            - push
          branch_patterns: 
            - pattern: main
            - pattern: main
              include: false
          cancel_previous_builds: true

              

### Expected results

it should successfully buld apk

### Actual results

./gradlew: No such file or directory

**Comment by himesh-cm** [comment]
Hi @jagdishjadeja , Please share a build id, so i can look into the same.

**Comment by himesh-cm** [comment]
Hi @jagdishjadeja, I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
